project-root/Assets/Scripts/Monsterspawner.cs

```diff
@@ -0,0 +1,84 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+using UnityEngine.AI;
+
+public class Monsterspawner : MonoBehaviour

```

Comment:
Since the "monster spawner" is not being used, it shouldn't even be in this pull request. Remove anything relating to the monster spawner, and just keep the navmesh spawner if that is what you plan on everyone using.